script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -195,7 +195,8 @@ There is no cure for one whose line of death has been cut.
 [sha:bb39322d841c9e2dcf1e69f6094b8363fae9cb95]{
 -- Page 90, Offset 30791.
 -- あの姫君でさえ蘇生するのに八百年の歳月と引き換えにするしかなかった。ただの人間に、“死”から逃れる術はない」
-That princess may have revived, but she had to give up 800 years of her life in exchange.
+// group pr suggestion: That princess may have revived after such an attack, but she had to give up 800 years of her life to do so. A mere human has no such means to escape from 'death'."
+That princess may have revived from it, but she had to give up 800 years of her life in exchange.
```

Comment:
Note was that people might not associate the revival with being 'killed'. Just the `from it` version seems fine to me.

Reply:
I'm not really sure I understand where the confusion might come from. I say stick with original line.

Reply:
Works for me